package.json

```diff
@@ -9,7 +9,6 @@
     "scripts": {
         "install-ci": "make npm",
         "bundle-config": "node scripts/bundleConfig.mjs && node scripts/bundleTrackers.mjs",
-        "bundle-se": "ddg2dnr smarter-encryption shared/data/smarter_encryption.txt shared/data/bundled/smarter-encryption-rules.json",
```

Comment:
This was intentional?

Reply:
Yea, we are turning `shared/data/smarter_encryption.txt` into `$(BUILD_DIR)/data/bundled/smarter-encryption-rules.json`. Since we can't do that directly from the package.json task, we had to create an intermediate file (which then ended up in the builds, which wasn't great). Better to just call ddg2dnr directly from the Makefile and create the file we want directly.